f1pystats/sprint_results.py

```diff
@@ -18,3 +18,21 @@ def get_driver_team(self):
     def get_driver_status(self):
         """Returns the driver sprint status"""
         return [i["status"] for i in self.results]
+    def get_driver_number(self):
+        """Returns the driver number"""
+        return [i["Driver"]["permanentNumber"] for i in self.results]
+    def get_laps(self):
+        """Returns the driver sprint laps"""
+        return [i["laps"] for i in self.results]
+    def get_driver_grid(self):
+        """Returns the driver sprint grid status"""
+        return [i["grid"] for i in self.results]
+    def get_driver_time(self):
+        """Returns the driver sprint time"""
+        time=[]
+        for i in self.results:
+            time.append(i.get("Time",{"time":"DNF"})['time'])
```

Comment:
Looks good. Will test it again later.

Can you explain what this particular line does?

Reply:
So for Dnf situation the api doesn't return the time or any data so i have added a default value of DNF

Reply:
Ah okay. Good catch there 👍🏽